src/api/app/services/scm_exception_handler.rb

```diff
@@ -0,0 +1,38 @@
+class SCMExceptionHandler
+  include ActiveSupport::Rescuable
+
+  rescue_from Octokit::AbuseDetected,
+              Octokit::AccountSuspended,
+              Octokit::BillingIssue,
+              Octokit::BranchNotProtected,
+              Octokit::Conflict,
+              Octokit::Forbidden,
+              Octokit::RepositoryUnavailable,
+              Octokit::NotFound,
+              Octokit::OneTimePasswordRequired,
+              Octokit::Unauthorized,
+              Octokit::UnavailableForLegalReasons,
+              Octokit::UnsupportedMediaType,
+              Octokit::CommitIsNotPartOfPullRequest,
+              Octokit::InstallationSuspended,
+              Octokit::SAMLProtected,
+              Octokit::TooLargeContent,
+              Octokit::TooManyLoginAttempts,
+              Octokit::UnverifiedEmail,
+              Octokit::InvalidRepository,
+              Octokit::PathDiffTooLarge,
+              Octokit::ServiceUnavailable,
+              Octokit::InternalServerError do |exception|
+    # FIXME: Inform users about the exceptions
```

Comment:
Can we at least log this so we have something to debug the problems of people?

Reply:
if we have the request id, couldn't we just grep the log file for that and get the stacktrace from there - or from errbit?

Reply:
It won't add anything to errbit. I'm not sure about the request id.

Reply:
I've added an error logger.

Reply:
my question was to @hennevogel. I wanted exactly to avoid the "Rails logger" calls everywhere. There should be better way to introspect code in 2021, other than custom stuff on error logger, right?

Reply:
Logging is a temporary measure, just to be able to debug this for people for now. In the future we are going to expose this to the users. Because it is *them* who can do something about this. we can't.

Reply:
@hennevogel Please have another look.